samples/app/FilamentApp.cpp

```diff
@@ -69,7 +71,9 @@ FilamentApp::~FilamentApp() {
 }
 
 void FilamentApp::run(const Config& config,SetupCallback setupCallback,
-        CleanupCallback cleanupCallback, ImGuiCallback imguiCallback, size_t width, size_t height) {
+        CleanupCallback cleanupCallback, ImGuiCallback imguiCallback,
+        RenderCallback render, PostRenderCallback postRender,
```

Comment:
Maybe call it `PreRenderCallback` to be symmetric with PostRender and to make it obvious that it occurs before `beginFrame`

Reply:
SG